package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "math-magicians",
   "version": "0.1.0",
+  "homepage": "https://danielochuba.github.io/Math-magicians",
```

Comment:
- [ ] When I run the application nothing happens and the page is blank, the default page is supposed to be the `Home` page but when I run the app the `Home` page doesn't render and this is because of the line I selected above so when we run the app the link that I select run as the home page by default. If you delete it everything will be fine, Kindly fix that 🙏🏾 

<img width="982" alt="Screenshot 2023-06-21 at 15 24 29" src="https://github.com/danielochuba/Math-magicians/assets/74501523/bc054d36-4349-4c3a-b6bd-03150f32aefa">